components/molecules/Menu.js

```diff
@@ -0,0 +1,81 @@
+import PropTypes from "prop-types";
+
+/**
+ * Menu component
+ */
+export function Menu(props) {
+  //Function for changing menu state
+  function onMenuClick() {
+    const menuButton = document.getElementById("menuButton");
+    document.getElementById("menuDropdown").classList.toggle("active");
+
+    menuButton.getAttribute("aria-expanded") === "true"
+      ? document
+          .getElementById("menuButton")
+          .setAttribute("aria-expanded", false)
+      : document
+          .getElementById("menuButton")
+          .setAttribute("aria-expanded", true);
+  }
+
+  return (
+    <nav
+      title="Menu"
+      className="layout-container lg:justify-end lg:flex"
+      data-cy="menu"
+    >
+      <div id="menuButtonDiv" className="pb-4 pt-2">
+        <button
+          id="menuButton"
+          onClick={onMenuClick}
+          className="text-h4 text-canada-footer-font focus:outline-none focus:ring-2 focus:ring-black"
+          aria-haspopup="true"
+          aria-expanded="false"
+        >
+          <span className="icon-menu" />
+          <span className="pl-3">{props.menuButtonTitle}</span>
+        </button>
```

Comment:
I looked up if the HTML nesting matters for this, but I didn't see any definitive info. 

Eg, the button says `"aria-expanded"` but the menu it opens is in another div, not immediately after the button. 

Most of the examples I've seen are more like this.
```
<div>
  <button aria-expanded="true">menu</button>
  <ul><li>Item 1</li><li>Item 2</li></ul>
</div>
```

Whereas ours looks like this:
```
<div>
  <div>
    <button aria-expanded="true">menu</button>
  </div>
  <div>
    <ul><li>Item 1</li><li>Item 2</li></ul>
  </div>
</div>
```

Not too worried about it since I didn't see any specific documentation about this, but just thought I'd mention. 

Nothing to change here.